### DISCUSSION TITLE
How to bundle components using Rsbuild

### DISCUSSION THREAD

**Discussion by congxiaobai** [web-infra-dev/rsbuild#3020] [Q&A] [answered]
打包React组件，输出为umd,esm等格式，请问如何配置

**Comment by chenjiahan** [comment, marked as answer]
We are developing Rslib to support library building.

Before that, you can try Modern.js Module, it is powered by esbuild. https://modernjs.dev/module-tools/en/index.html

**Comment by congxiaobai** [comment]
When will  Rslib be released?

**Reply by chenjiahan** [reply]
Rslib is still in an early stage and we will open source it soon

**Reply by ruibaby** [reply]
> Rslib is still in an early stage and we will open source it soon

Looking forward to it~

**Reply by chenjiahan** [reply]
The Rslib repository is now public: https://github.com/web-infra-dev/rslib
